src/css/styles.css

```diff
@@ -582,6 +582,37 @@ a.multiple-recordings__action-button {
   cursor: help;
 }
 
+/*************************** LINGUISTIC BREAKDOWN ***************************/
+
+/**
+ * BLOCK linguistic-breakdown
```

Comment:
Throughout, 'breakdown' is a plain English word where I'd rather use 'analysis', simply because 'breakdown' would better apply to the (inflectional/derivational) morpheme breakdown, so splitting a word by its morpheme boundaries, whereas the tags/features are 'analytical' characteristics of the entire word.

Perhaps something for later.